docs/guide/index.md

```
@@ -18,40 +18,57 @@
+  const client = new WebMcpClient()
+  await client.connect(clientTransport)
+  // 这个 sessionId 是 Web 应用与 WebAgent 服务建立连接后，由 WebAgent 服务生成的，用来唯一标识被操控的 Web 应用（被控端）
+  const { sessionId } = await client.connect({
+    agent: true,
+    url: 'https://agent.opentiny.design/api/v1/webmcp-trial/mcp'
+  })
+})
 ```
 
-**第三步：创建 WebMcpClient ，并与 WebAgent 连接**
+**第三步：创建 WebMcpServer ，并与 ServerTransport 连接**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)